core/src/main/java/org/aerogear/mobile/core/configuration/ServiceConfiguration.java

```diff
@@ -35,28 +76,19 @@ public String getProperty(String name) {
         return properties.get(name);
```

Comment:
Does this compile?
`properties` is a `Map<String,String>`, so `properties.get(name)` returns a string, however the method returns a `Map<String,String>`,

Reply:
Perhaps that was the way GitHub was excluding some sections. If I expand this block this is the code

```java
    public Map<String, String> getProperties() {
        return Collections.unmodifiableMap(properties);
    }

    public String getProperty(String name) {
        return properties.get(name);
    }
```

Reply:
My bad ;)